user_sync/rules.py

```diff
@@ -291,24 +303,29 @@ def is_selected_user_key(self, user_key):
         return True
     
     def process_orphaned_dashboard_users(self):
-        remove_user_key_list = self.remove_user_key_list
+        orphan_user_key_list = self.orphan_user_key_list
             
         options = self.options
-        remove_list_output_path = options['remove_list_output_path']
+        orphan_list_output_path = options['orphan_list_output_path']
         remove_nonexistent_users = options['remove_nonexistent_users']
+        delete_nonexistent_users = options['delete_nonexistent_users']
         
         max_deletions_per_run = options['max_deletions_per_run']
         max_missing_users = options['max_missing_users']
 
-        orphaned_dashboard_users = list(self.iter_orphaned_dashboard_users(self.options['managed_identity_types']))
-        self.logger.info('Orphaned users to be removed: %s', [self.get_dashboard_user_key(dashboard_user) for dashboard_user in orphaned_dashboard_users])
+        if self.process_orphaned_accounts_for_delete:
+            orphaned_dashboard_users = list(self.iter_orphaned_dashboard_users())
```

Comment:
No, that's not it.  The `managed_identity_types` option in the config file, which defaults to `[federatedID, enterpriseID]`, is what controls which kinds of accounts can be removed/deleted.  So if you alter your config to add `adobeID` to that list then this code will also delete the Adobe IDs.  That's why we make the back end smart about just doing removeFromOrg on Adobe IDs if the client asks for them to be deleted.